### DISCUSSION TITLE
Which SLiRP to use?

### DISCUSSION THREAD

**Discussion by OBattler** [86Box/86Box#1174] [Ideas]
I was told that there's problem getting our current SLiRP (based on libslirp) to compile with MSVC, so maybe we should get waltje's slirp to work instead as that's known to compile correctly with MSVC.

**Comment by ghost** [comment]
The current implementation of SLiRP is better than any around us. So I disagree for using it.

**Reply by OBattler** [reply]
How is it better when it has problems compiling with MSVC while waltje's doesn't?
